docs/features/search/getting-started.md

```diff
@@ -163,9 +164,14 @@ export default async function createPlugin(
     searchEngine,
   });
 
+  const catalogIndexSchedule = env.scheduler.createScheduledTaskRunner({
```

Comment:
Trying to grok the implications of collators sharing a schedule (seems like it's totally fine because the coordination is based on the task ID, which is derived from the collator type?)...

Wondering if it would clarify that if we encouraged schedule naming to reflect the frequency, rather than the collator?  e.g. `const every10MinutesSchedule = ...` or similar